bigframes/session/executor.py

```diff
@@ -385,12 +386,24 @@ def peek(
         if not tree_properties.can_fast_peek(plan):
             msg = "Peeking this value cannot be done efficiently."
             warnings.warn(msg)
+        if use_explicit_destination is None:
+            use_explicit_destination = bigframes.options.bigquery.allow_large_results
+
+        job_config = bigquery.QueryJobConfig()
+        # Use explicit destination to avoid 10GB limit of temporary table
+        if use_explicit_destination:
+            destination_table = self.storage_manager.create_temp_table(
```

Comment:
I don't think it needs to be a full table, right? We should be able to avoid the `tables.create` call. `_random_table` seems more appropriate (https://github.com/googleapis/python-bigquery-dataframes/blob/a745290896e34a0bb25ae023ab17fb04d2c5887f/bigframes/session/temp_storage.py#L70)

Reply:
Ah, good point.